src/xradio/image/_util/_fits/xds_from_fits.py

```diff
@@ -507,32 +510,7 @@ def _get_time_values(helpers):
 
 
 def _get_pol_values(helpers):
-    # as mapped in casacore Stokes.h
```

Comment:
This mapping already exists in `xradio.measurement_set._utils._utils.stokes_types`, so the list was replaced with the mapping table from there.